src/software/ai/hl/stp/evaluation/possession.h

```diff
@@ -1,6 +1,5 @@
-#include "software/ai/world/ball.h"
-#include "software/ai/world/field.h"
-#include "software/ai/world/team.h"
+#pragma once
+#include "software/ai/world/world.h"
```

Comment:
We should add tests for all the functions in this file

Reply:
Going to create an issue to test `teamPassInProgress`